frontends/mit-open/src/page-components/SearchDisplay/SearchDisplay.tsx

```diff
@@ -489,19 +513,32 @@ const SearchDisplay: React.FC<SearchDisplayProps> = ({
   setParamValue,
   clearAllFacets,
   toggleParamValue,
-  patchParams,
   showProfessionalToggle,
+  setSearchParams,
 }) => {
+  const [searchParams] = useSearchParams()
+  const activeTab =
+    TABS.find((t) => t.name === searchParams.get("tab")) ??
```

Comment:
Previously the active tab was based on `resource_type` query parameter.

Now the tabs are:
- All `tab="all"` ... `resource_type=[]`
- Courses `tab="courses"` ... `resource_type=["course"]`
- Programs `tab="programs"` ... `resource_type=["program"]`
- Learning Materials `tab="learning-materials"` ... `resource_type=["video", "podcast", "podcast_episode", "video_playlist", "learning_path"]`.

I was a little torn about whether to infer the tab based on `resource_type` query param, or use a separate `tab` param (which is what I ended up doing).

Using a separate tab really simplifies URLs like `href: querifiedSearchUrl({ tab: "learning-materials" }),` in the navbar. 

My thought is that **in the future** when we add a "resource_type" sidebar facet (for the LM tab), the URLs would look like `?tab=learning-materials&resource_type="video"`.